test/e2e/network/dns.go

```diff
@@ -70,9 +70,9 @@ var _ = SIGDescribe("DNS", func() {
 	})
 
 	// [LinuxOnly]: As Windows currently does not support resolving PQDNs.
+	// TODO: Remove checking google.com name resolution to make this E2E eligible for Conformance. Remove NodeOSDistroIs("windows") check, as it is not needed for [LinuxOnly] test.
 	ginkgo.It("should resolve DNS of partial qualified names for the cluster [LinuxOnly]", func() {
 		// All the names we need to be able to resolve.
-		// TODO: Spin up a separate test service and test that dns works for that service.
```

Comment:
Removed TODO: in favouring https://github.com/kubernetes/kubernetes/pull/74977#discussion_r303699978